### ISSUE TITLE
Server side rendering broken

### ISSUE THREAD

**Issue by brechtpm** [yoshuawuyts/choo#441]
### Expected behavior

App code can run in node.js

### Actual behavior

Running choo v5.0.2 in node.js gives this error

```
module.js:471
    throw err;
    ^

Error: Cannot find module 'document-ready'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/savelbr/Projects/forks/choo/index.js:1:83)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

### Steps to reproduce behavior

Very basic choo app to reproduce:

```
// app.js

var choo = require('choo')
var app = choo()
module.exports = app
```

Running this with `node app.js` gives the error above.

**Comment by brechtpm** [comment]
To solve this, I think wrapping the `var documentReady = require('document-ready')` at the top of index.js should suffice. Because if I just delete it, it does work as expected in node (though obviously not anymore in the browser).

Not exactly sure how to wrap it though `typeof process === 'undefined'` doesn't seem to work, and neither do a couple of other things I tried.

**Comment by yoshuawuyts** [comment]
Fixed in `v5.0.4`
